### DISCUSSION TITLE
Aftershock Coming Attractions by the Maintainers

### DISCUSSION THREAD

**Discussion by Maleclypse** [CleverRaven/Cataclysm-DDA#45975] [Ideas]
The Aftershock Maintainers have decided to list out some things that are on our lists to accomplish in the next few years. This way we can discuss among ourselves about ideas and flesh them out further or change them as the setting may need.  If you see something on this list from @John-Candlebury , @Mom-Bun, or myself (Maleclypse) and you decide to take a crack at adding it to `Aftershock` that's fair as in some cases the items on this list may take years.  To start with I'm going to list out my items and sentence or two about them from my notes.

- Additional PrepNet additions.  One in my mind is a sort of fireteam point at an intersection on a road.  These are taking a back seat as we shift the lore of Aftershock from cyberpunk to extra-solar. As they will likely be a rogue colonizing group trying to squat a planet.

- Aftershock specific newspaper snippets and survivor's notes.  These are crucial and can be added in any amount to add depth and lore to the universe.

- more mi-go biotech recipes, these will probably be available through Whately family npcs.

- Additional uplifted bloodlines (genetically modified animals with some level of human grade intelligence) some of these animals will be human shaped such as the bearmen Schwarz Walders and the octopoid merfolf Cecaelians while others will be of more limited intelligence and in animal forms.  Domestic cats with parrot attacks of "How was your day?" "I deserve pets" "Feed me human!" are a possible idea.

- Items using unpack and the jsonified internal furnace functions to simulate high tech instafab items.

-Conservatories that contain rare grown foodstuffs from Earth not suited to the new world's climate many genemodded look to existing seeds from Aftershock green room nests.  Also sometimes with fauna and flora imported in from other extrasolar colonies.  Did any of it survive the collapse of the colony?

-A hospital faction camp upgrade that allows the PC with tons of afs_scrap to build their own jury rigged autodoc.  This is a low quality autodoc that will still require anesthesia and be dangerous but convenient to use. 

- Crafting software items to store on USB sticks that will be an intermediary step in crafting AI driven hypertech items, such as the autodoc. These items will combine with AI cores from robots to go into whatever new AI needing item is created.

-supertech yeasts that grow into drugs, including some supertech drugs

-redesign the fusion blaster as a light gas gun (see wikipedia) 

-what happened stories for common Aftershock NPCs. How did they get here from civilized space or were they born here after their ancestors were left behind?

- mi-go island/underwater base to go with the deep one migo variants

-Kinetic weave loom makes kevlar sheets out of afs_scrap utilizes an AI component in construction as referenced above.  

-specialized clothes and armors (maybe a gibson gear company in honor of our roots)

This is probably a good start.  Looking forward to hearing from others.

**Comment by Mom-Bun** [comment]
I'm currently not working on anything that pops out as "cool" or "interesting" right now as I very much want to focus on the groundwork that'll hopefully make the transition to the extra-solar colony much easier, but in the future I do have some ideas of what I'd love to include:

In the short term, IE, what I'm working on right now:

- Currently, I am working on harvest groups, recipes, general infrastructure that'll allow making new alien fauna, moxphores, and anything in between much much easier. Alongside these changes, I will want to include more Moxphores, both alien and human, bio-machines, and alien fauna to give the world a more nice feeling.
- I'll want to work on buildings as well, adding furniture, item groups, palettes, and spawn groups that'll not only help make adding new buildings in the future easier, but will offer a groundwork to make the transition somewhat easier.
- New terrains, furniture, and items, anything that'll make sense for the aforementioned building inclusions.

On note for alien fauna, I'm going to take my time on this I think, as I want the aliens I add to be... interesting, but one fun thing I'm trying to do is use the snippets for alien refuse, I don't know how effective this'll be, but it'll possibly add some interesting flavor, and if it works, it means we can use snippets for the rotten old food in the colonies as well.

Another fun short thing I want to tackle is re-working the energy saber, as right now its not the most interesting, with little lore put into it, and I'd love to re-work it as a tactical baton, where the non-lethal mode is the "off" mode, and the lethal mode is "on" with the lethal mode in this case being a red hot blade folded out. I'd want to add in mentions in the description of how this was used in the enforcement and self defense for corporate enforcers on off-world colonies.

In the more long term of what I'd love to include:
- A new faction based around middle managers for a megacorp that get shunted off to a backwater, and end up in a dead end job due to it. 
- Possibly a MoxPhore mutation line? I'll have to think on this more, as it would need to be done correctly, as in a normal scenario a MoxPhore mutation line would be a game over, so it would be a temporary thing, requiring it to be cured before you can leave.
- Expanding on lore more as we shift to new lore, like Maleclypse outlined before, stuff like newspaper, flyers, any real snippets, plus some stuff to expand on the aliens, maybe even snippets for miscellaneous tech?
- Some new NPCs and quests, 100%
- More aliens, monsters, locations, items, and anything under the sun, always

**Comment by kevingranade** [comment]
Do y'all have/want permissions to make a project board for Aftershock?

**Reply by Maleclypse** [reply]
I would absolutely utilize that if you are offering.  :)

**Reply by Mom-Bun** [reply]
Same here, a project board would be very useful

**Reply by John-Candlebury** [reply]
I have project permissions, but Maleclypse and Mom-Bun having them too would be helpful

**Comment by kevingranade** [comment]
Invites sent

**Reply by Maleclypse** [reply]
Thank you!

**Comment by actual-nh** [comment]
Re having CBM "loadouts" that are easily changeable:

- Artificial hands/forearms/arms, feet/calves/legs that can be changed out.
- For neurological stuff (reflexes, int boosts...), limited-capacity processing with different programs to load in.

**Reply by Maleclypse** [reply]
I think we'll see what the new limb stuff looks like and what we can do with that in the 0.F experimental.
